tty/uart16550/uart16550.c

```diff
@@ -242,14 +242,16 @@ static void poolthr(void *arg)
 
 static int _uart_init(uart_t *uart, unsigned int uartn, unsigned int speed)
 {
-	unsigned int divisor = 115200 / speed;
+	unsigned int divisor;
 	libtty_callbacks_t callbacks;
 
-	int err = uarthw_init(uartn, uart->hwctx, sizeof(uart->hwctx));
+	int err = uarthw_init(uartn, uart->hwctx, sizeof(uart->hwctx), &divisor);
 	if (err < 0) {
 		return err;
 	}
 
+	divisor /= 16 * speed;
```

Comment:
Magic number `16`. I will appreciate a comment.

Reply:
Clock is divided internally by 16 (rx oversampling). Pls add comment if you find it necessary